README.md

```diff
@@ -225,6 +226,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 
 ### • File Manager
 
+* [**Amaze File Manager**](https://github.com/TeamAmaze/AmazeFileManager) <sup>**[[F-Droid](https://f-droid.org/packages/com.amaze.filemanager/)]**</sup>
```

Comment:
Amaze is no longer FOSS unfortunately and was removed therefore (see #217).

Reply:
Oops, should've checked

Reply:
No problem :-)